src/Packetery/Module/WpdbTracyPanel.php

```diff
@@ -0,0 +1,103 @@
+<?php
+/**
+ * Class WpdbTracyPanel
+ *
+ * @package packetery
+ */
+
+declare( strict_types=1 );
+
+
+namespace Packetery\Module;
+
+use PacketeryLatte;
+use PacketeryTracy;
+
+/**
+ * Class WpdbTracyPanel
+ *
+ * @package packetery
+ */
+class WpdbTracyPanel implements PacketeryTracy\IBarPanel {
+
+	/**
+	 * WpdbAdapter.
+	 *
+	 * @var WpdbAdapter
+	 */
+	private $wpdbAdapter;
+
+	/**
+	 * Latte engine.
+	 *
+	 * @var PacketeryLatte\Engine
+	 */
+	private $latteEngine;
+
+	/**
+	 * Constructor.
+	 *
+	 * @param WpdbAdapter           $wpdbAdapter WpdbAdapter.
+	 * @param PacketeryLatte\Engine $latteEngine Latte engine.
+	 */
+	public function __construct(
+		WpdbAdapter $wpdbAdapter,
+		PacketeryLatte\Engine $latteEngine
+	) {
+		$this->wpdbAdapter = $wpdbAdapter;
+		$this->latteEngine = $latteEngine;
+	}
+
+	/**
+	 * Get queries.
+	 *
+	 * @return \Generator
+	 */
+	private function getPacketeryQueries(): \Generator {
+		$queries = $this->wpdbAdapter->getWpdbQueries();
+
+		$count = 0;
+		foreach ( $queries as $queryInfo ) {
+			[ $query, $timeSpent, $funcList ] = $queryInfo;
+
+			if ( false === strpos( $funcList, 'Packetery\\' ) && false === strpos( $funcList, '/plugins/packeta/' ) ) {
+				continue;
+			}
+
+			yield [
+				'query'     => $query,
+				'timeSpent' => $timeSpent * 1000,
```

Comment:
Phpdoc ve wpdb mi tvrdí: @type float  $1 Total time spent on the query, in seconds. Jsme si jisti výpočtem, tedy, že vstup jsou opravdu mikrosekundy?

Reply:
Předávají se milisekundy